proxy/envoy/testdata/lds-httpproxy.json.golden

```diff
@@ -33,7 +33,41 @@
            "source.ip": "10.1.1.0",
            "source.uid": "kubernetes://v0.default"
           },
-          "quota_name": "RequestCount"
+          "quota_name": "RequestCount",
+          "v2": {
+           "controlConfigs": {
+            "hello.default.svc.cluster.local": {
+             "mixerAttributes": {
+              "attributes": {
+               "destination.service": {
+                "stringValue": "hello.default.svc.cluster.local"
+               }
+              }
+             }
+            }
+           },
+           "defaultDestinationService": "hello.default.svc.cluster.local",
+           "forwardAttributes": {
+            "attributes": {
+             "source.ip": {
+              "bytesValue": "AAAAAAAAAAAAAP//CgEBAA=="
```

Comment:
Is "bytes" type in JSON base64 encoded?

Reply:
Yes - see https://developers.google.com/protocol-buffers/docs/proto3#json.

Reply:
Clarification - the jsonpb encoding of bytes type is base64 encoded. The `v2` object in the mixerclient JSON object is jsonpb encoded but the parent JSON is normal json encoded.